sentry_auth_github/client.py

```diff
@@ -35,7 +35,7 @@ def _request(self, path, access_token):
                 headers=headers,
             )
         except RequestException as e:
-            raise GitHubApiError(unicode(e), status=e.status_code)
+            raise GitHubApiError(unicode(e), status=getattr(e, 'status_code', None))
```

Comment:
What about returning `0` here instead? Seems sane that we'd expect an integer to be returned here for status. I could imagine this attempting to be used in string formatting like `'%d' % e.status`, and the `None` would blow up.